### PR TITLE
FIX Tidy extension and cli fix for tests

### PR DESCRIPTION
**Pull request by chillu**
Wrap doesn't actually wrap in the tidy extension.
This causes tests to be flakey, for example some of `FormTest` fails
if you happen to have `ext-tidy` installed (which is the default on most systems).
This happened to me on PHP 7.4.19 with tidy 5.6.0 (OSX Homebrew).
Note that the tests aren't failing in Travis right now.

You'd expect `wrap => 0` to be honoured. It's documented as an integer
in the tidy docs: https://api.html-tidy.org/tidy/quickref_5.6.0.html#wrap.

Even tracked this down to the PHP source which appears to be doing the right thing:
https://github.com/php/php-src/blob/master/ext/tidy/tidy.c#L300

There's a bug from 2018 against PHP 7.2.8 which was closed as "not a bug" without comment:
https://bugs.php.net/bug.php?id=76683

You can see the behaviour in action in the following test.

```
<?php
$html = '<p>a really long string which should not be wrapped</p>';

echo "## With tidy extension" . PHP_EOL;
$tidy = new tidy();
$tidy->parseString(
    $html,
    [
        'output-xhtml' => true,
        'numeric-entities' => true,
        'wrap' => 0,
    ],
    'utf8'
);
$tidy->cleanRepair();
echo $tidy . PHP_EOL;

echo "## With tidy cli" . PHP_EOL;
$cmd = sprintf("echo " . escapeshellarg($html) . " | tidy --force-output 1 -n -q -utf8 -asxhtml -w 0 2> /dev/null");
echo shell_exec($cmd);
```

Long story short, setting it to 99999 fixes the issue.

### CODE REVIEW

**Review by dhensby — APPROVED** [review]

**Review by NightJar — APPROVED** [review]
Nice sleuthing.
I always cringe at `999999` recurring used in places (normally `css` `z-index`) - but there's a legitimate reason here and you've left a minor comment, and the commit message explains more :) A rock and a hard place, as they say.

A nice, good long commit message filled with the context it needs to provide to be understood ❤️ I love to see it.

**Comment by ntd** [comment]
> You'd expect `wrap => 0` to be honoured.

FYI this seems to be is a [bug in tidy-html5](https://github.com/htacg/tidy-html5/pull/705) introduced in 5.6.0 and resolved in 5.7.9. The high `wrap` value workaround is the "official" fix if you target those versions.

**Comment by chillu** [comment]
Awesome thanks for tracking that down @ntd!
